src/main/webapp/WEB-INF/tags/instructor/comments/pagination.tag

```diff
@@ -0,0 +1,14 @@
+<%@ tag description="Course Pagination" %>
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
+<ul class="pagination">
+    <li><a href="${data.previousPageLink}">�</a></li>
```

Comment:
I see a special character in my diff view.
![image](https://cloud.githubusercontent.com/assets/1673303/8159887/932f11bc-139d-11e5-88d2-ed284e2d70c3.png)